components/canvas/webgl_thread.rs

```diff
@@ -830,6 +831,25 @@ impl WebGLThread {
                 "... rebound framebuffer {}, new back buffer surface is {:?}",
                 framebuffer_object, id
             );
+
+            // Tell Webrender to invalidate the caches for this image
+            let info = self.cached_context_info.get_mut(&context_id).unwrap();
+            let has_alpha = data
+                .state
+                .requested_flags
+                .contains(ContextAttributeFlags::ALPHA);
+            let image_buffer_kind = current_wr_image_buffer_kind(&self.device);
+            Self::update_wr_external_image(
+                &mut self.webrender_api,
+                self.webrender_doc,
+                size.to_i32(),
+                has_alpha,
+                context_id,
+                info.image_key,
+                image_buffer_kind,
+            );
+
+            debug_assert_eq!(data.gl.get_error(), gl::NO_ERROR);
```

Comment:
Is this new assertion related?

Reply:
No, you are correct. There are no new OpenGL calls before this, so this assertion is not needed. Removed in [ec5c9df](https://github.com/servo/servo/pull/30390/commits/ec5c9df530355d1d406a497d9e4c9b538213f18f)